### ISSUE TITLE
Should not use vite bundle node package

### ISSUE THREAD

**Issue by gweesin** [epicmaxco/vuestic-ui#4216]
**Vuestic-ui version:** 1.9.8

### Description

When I checked the code with #4215 , I found `@vuestic/tailwind` bundle packages by Vite. 

It's really not recommended by https://github.com/vitejs/vite/issues/13926

**Comment by m0ksem** [comment]
Yeah. We should use tsup instead. But for now, vite don't cause any issues except we need to exclude node internals.
